WooCommerce/Resources/release_notes.txt

```diff
@@ -1,4 +1 @@
-- [*] Handle sites configured with `http` siteAddress. [https://github.com/woocommerce/woocommerce-ios/pull/16279]
-- [*] Use tabs cache state to display conditional tabs initially. [https://github.com/woocommerce/woocommerce-ios/pull/16292]
-- [*] Handle sites configured with `http` siteAddress.
-- [*] Fix: Unable to dismiss keyboard when editing Product Title. [https://github.com/woocommerce/woocommerce-ios/pull/16288]
\ No newline at end of file
+This update improves app stability and usability. We’ve enhanced compatibility for stores using HTTP site addresses, optimized how tabs load based on saved states, and fixed an issue that prevented dismissing the keyboard when editing product titles.
```

Comment:
I believe the tabs cache fix is still under feature flag as CIAB sites are not public right now. Can we remove that from the release note? 

cc @RafaelKayumov

Reply:
On a second thought I suppose this affects the POS tab, so this should be good. Please feel free to ignore my comment if that's the case.